docs/cloud/registry.md

```diff
@@ -0,0 +1,29 @@
+# Private Docker registry
```

Comment:
I think this is a Cortex Core page? Let's discuss (with Vishal)

Reply:
I think it makes sense for the private registry to be in the cloud section.